src/screens/EmptyScreen.tsx

```diff
@@ -2,7 +2,13 @@ import React from "react";
 import { Dimensions, SafeAreaView, StyleSheet, View, Text } from "react-native";
```

Comment:
Esse arquivo está sendo usado em algum lugar? senão acho que podemos remover já, era mais pra ajudar no início....